### PR TITLE
add a sprite for the Scram drive

### PR DESCRIPTION
**Pull request by Brick63**
this simply adds in a new sprite to differentiate the scram drive and Hyperdrive
assets:
[scram drive assets.zip](https://github.com/endless-sky/endless-sky/files/1605103/scram.drive.zip)

### CODE REVIEW

File: copyright [hunk 1]
```diff
@@ -260,8 +260,10 @@ Files: images/outfit/tactical?scanner*
 Copyright: Zachary Siple
 License: CC-BY-SA-4.0
 
-Files: images/outfit/korath?rifle*
-Copyright: Evan Fluharty (Evanfluharty@gmai.com)
+Files:
+images/outfit/korath?rifle*
```
Comment (tehhowch) [inline copyright]: Under `Files:` the lines should be indented (see the other entries)

Comment (Brick63) [reply copyright]: okay

File: copyright [hunk 2]
```diff
@@ -260,8 +260,10 @@ Files: images/outfit/tactical?scanner*
 Copyright: Zachary Siple
 License: CC-BY-SA-4.0
 
-Files: images/outfit/korath?rifle*
-Copyright: Evan Fluharty (Evanfluharty@gmai.com)
+Files:
+ images/outfit/korath?rifle*
```
Comment (KiLEdEnNis) [inline copyright]: just wondering what this line is doing here & not in the merged PR

Comment (Brick63) [reply copyright]: i forgot to put the l at the end of gmail

**Comment by Bladewood** [comment]
Why is it called `scramdrive` and not `scram drive`?

**Comment by Brick63** [comment]
i named the files and i cant change the names as far as i know, and does it really matter

**Comment by Brick63** [comment]
if it does then ill work to fix it but if not then ill leave it be

**Comment by Bladewood** [comment]
Trivial as it may be, yes, it matters.
According to [the Pro Git book](https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository), all you have to do is rename the file and commit again.

**Comment by Brick63** [comment]
well okay

**Comment by Brick63** [comment]
there fixed

File: copyright (hunk 2 above)
Comment (Brick63) [reply copyright]: also it was in the PR i just moved it to follow the setup

**Pull request merged**